StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+SwiftUI.swift

```diff
@@ -134,17 +137,43 @@ import Combine
         embeddedPaymentElement?.testHeightChange()
     }
 #endif
+    
+    private func calculateAndPublishHeight() {
+        guard let embeddedPaymentElement else { return }
```

Comment:
nit: take an `embeddedPaymentElement` and we can remove the need to think (eg wonder how we end up in that situation, if we should assert, etc) about it being nil.

Reply:
Ah, nice!